.pre-commit-hooks.yaml

```diff
@@ -4,3 +4,8 @@
   entry: gitleaks protect --verbose --redact --staged
   language: golang
   pass_filenames: false
+- id: gitleaks-docker
+  name: Detect hardcoded secrets
+  description: Detect hardcoded secrets using Gitleaks
+  entry: zricethezav/gitleaks:v8.5.2 protect --verbose --redact --staged
```

Comment:
@foolioo I can't guarantee that I'll update this value every release. I'll try my best and if I miss any then it'll be a good opportunity and low effort way for folks to contribute to the project

Reply:
I could build some process around auto-updating this value but I'm stretched a bit thin

Reply:
@zricethezav Maybe we could use [`${{ github.event.release.tag_name }}`](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-example-36=) or [`${{ steps.meta.outputs.tags }}`](https://github.com/zricethezav/gitleaks/blob/1ddc924a676ec33df112a74f0b22e77d4c919c57/.github/workflows/release.yml#L48) in the [release](https://github.com/zricethezav/gitleaks/blob/master/.github/workflows/release.yml) workflow.

If during the release workflow execution we update the version within the `.pre-commit-hooks.yaml` and push the changes to the repository, I am unsure whether it will be a part of the release.

Do you have any idea?

Reply:
e.g. the gitleaks-docker hook does not include in the current release https://github.com/zricethezav/gitleaks/blob/v8.5.2/.pre-commit-hooks.yaml